src/NuGet.Services.EndToEnd/PushTests.cs

```diff
@@ -55,5 +55,28 @@ public PushTests(PushedPackagesFixture pushedPackages, ITestOutputHelper logger)
 
             await _clients.V3Search.WaitForPackageAsync(package.Id, package.Version, _logger);
         }
+        
+        [SemVer2Fact]
+        public async Task NewSemVer2PackageIsFiltered()
+        {
+            // Arrange
+            var package = await _pushedPackages.PushAsync(PackageType.SemVer2Prerelease, _logger);
+
+            // Act
+            // Wait for package to become available
+            await _clients.V3Search.WaitForPackageAsync(package.Id, package.Version, _logger);
+
+            var shouldBeEmptyV3 = await _clients.V3Search.Query($"q={package.Id}");
```

Comment:
`&prerelease=true` here. The only different between "should be empty" and "should not be empty" is the `semVerLevel=2.0.0`.